### PR TITLE
Make kubernetes overcommitment factors configurable

### PR DESCRIPTION
**Pull request by gregorjerse**
via the settings variable.

EXP-617
EXP-565

### CODE REVIEW

File: tests/settings.py [hunk 1]
```diff
@@ -117,7 +117,13 @@
 #:   {"batch": {"nodegroup": ["compute"]}, "interactive": {"nodegroup": ["interactive"]}}
 #: The keys in the scheduling_class dictionary must correspond to the lower
 #: case name of the process scheduling class.
-FLOW_KUBERNETES_AFFINITY = os.environ.get("RESOLWE_KUBERNETES_AFFINITY", None)
+FLOW_KUBERNETES_AFFINITY = None
```
Comment (dblenkus) [inline tests/settings.py:120]: Why is this changed in this commit?

**Review by dblenkus — COMMENTED** [review]
Add a change log.

File: tests/settings.py (hunk 1 above)
Comment (gregorjerse) [reply tests/settings.py:120]: You are right, it deserves a commit of its own. 

I made a stupid mistake to read a JSON encoded variable from environment directly, which would not work. Since we do not really need this in Resolwe I decided to add a default value in the settings file.

**Review by dblenkus — APPROVED** [review]
